sources/packages/api/src-sql/User/Create-User.sql

```diff
@@ -0,0 +1,26 @@
+-- Users Table
+CREATE TABLE IF NOT EXISTS users ( 
+  id SERIAL PRIMARY KEY,
+  user_name VARCHAR(300) NOT NULL UNIQUE,
+  email VARCHAR(300) NOT NULL,
+  first_name VARCHAR(100) NULL,
+  last_name VARCHAR(100) NULL,
+  created_at timestamp without time zone NOT NULL DEFAULT now(),
+  updated_at timestamp without time zone NOT NULL DEFAULT now()
+);
+
+-- Indices
+-- # id index
+CREATE INDEX IF NOT EXISTS idx_user_id_20210115_5_00 ON users (id);
```

Comment:
The index names are using "user" and the DROP INDEX is using the "users" in the Drop-User.sql file.
Please rename it accordingly.

Reply:
Fixed, keep it as table name